views/scss/xml-editor.scss

```diff
@@ -0,0 +1,9 @@
+.section-container {
+  .content-container{
+    max-width: calc(100% - 280px)!important;
```

Comment:
Is the `!important` required? It would be better to avoid using it.
The same for the other one line 7.

Reply:
Important statement is removed. 
TBH I would avoid adding styles at all, but the problem is `content-container` is growing with a child content.
I would change it on the layout level to avoid horizontal scrolling on all pages but it's a bit scared